examples/camera-app/linux/include/media-controller/default-media-controller.h

```diff
@@ -39,8 +40,12 @@ class DefaultMediaController : public MediaController
     // if the transport is ready.
     void DistributeVideo(const char * data, size_t size, uint16_t videoStreamID) override;
     void DistributeAudio(const char * data, size_t size, uint16_t audioStreamID) override;
+    void SetPreRollLength(Transport * transport, uint16_t PreRollBufferLength) override;
 
 private:
-    std::vector<Connection> mConnections;
-    std::mutex mConnectionsMutex;
+    // TODO: Look into ways to fetch from camera-device
+    PreRollBuffer preRollBuffer{ 4096 };
+    std::vector<Connection> connections;
```

Comment:
why rename mConnections to connections? We should use m prefix for each class members so that we can distinguish them with local varibles